train.py

```diff
@@ -232,6 +232,12 @@ def loss_fn(pred, labels):
     model_config.max_seq_len = job_config.training.seq_len
 
     logger.info(f"Building {model_name} {job_config.model.flavor} with {model_config}")
+    logger.info(
+        f"Detailed training config, local_batch_size: {job_config.training.batch_size}, "
+        f"seq_len: {job_config.training.seq_len}, "
+        f"total_steps: {job_config.training.steps}({job_config.training.warmup_steps}), "
+        f"activation_checkpoint: {job_config.activation_checkpoint.mode}"
```

Comment:
we can remove AC here, as it has been logged at https://github.com/pytorch/torchtitan/blob/main/torchtitan/parallelisms/parallelize_llama.py#L449